### PR TITLE
[ACR] Enable to run black in ci

### PR DESCRIPTION
**Pull request by YalinLi0312**

### CODE REVIEW

**Comment by YalinLi0312** [comment]
Hi @msyyc , can you help to run black on mgmt package? Or do you know who can help if you are not the best contact?

**Comment by msyyc** [comment]
> Hi @msyyc , can you help to run black on mgmt package? Or do you know who can help if you are not the best contact?

Mgmt python sdk is generated automatically and have been `blacked`. Maybe the difference is caused by the different black configuration. Why do you want to open format check in CI?

**Comment by YalinLi0312** [comment]
> > Hi @msyyc , can you help to run black on mgmt package? Or do you know who can help if you are not the best contact?
> 
> Mgmt python sdk is generated automatically and have been `blacked`. Maybe the difference is caused by the different black configuration. Why do you want to open format check in CI?

We run black with `-l 120` option to allow lines up to 120 characters.

File: sdk/containerregistry/ci.yml [hunk 1]
```diff
@@ -30,6 +30,7 @@ extends:
   parameters:
     TestProxy: true
     ServiceDirectory: containerregistry
+    ValidateFormatting: true
```
Comment (msyyc) [inline sdk/containerregistry/ci.yml:33]: I have tried to use `black -l 120` to format mgmt SDK code but CI still fails. Maybe there is still difference. Now that you have formatted `azure-containerregistry`, there is no need to check it in CI.

Comment (YalinLi0312) [reply sdk/containerregistry/ci.yml:33]: Thanks for your update! `setup.py` is not blacked, I just pushed the updates, hope ci will pass. Btw, here's how we run black: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/eng_sys_checks.md#black

**Review by mccoyp — APPROVED** [review]

**Pull request merged**